### PR TITLE
clean up RollingMethods

### PR DESCRIPTION
**Pull request by grimreaper**

### CODE REVIEW

File: code/src/java/pcgen/core/RollingMethods.java [hunk 1]
```diff
@@ -436,28 +433,13 @@ else if ((param instanceof Vector) && (curNumberOfParameters == 3))
...
-                }
-                else
+                if ((dRolls > Integer.MAX_VALUE) || (dFaces > Integer.MAX_VALUE))
                 {
-                    result = roll(iRolls, iFaces, keep);
+                    throw new ParseException("Values greater than " + Integer.MAX_VALUE + " not allowed.");
                 }
+                int iRolls = Math.round(((Double) numberOfRolls).intValue());
+                int iFaces = Math.round(((Double) faces).intValue());
+                double result = (keep == null) ? RollingMethods.roll(iRolls, iFaces, numToKeep, reroll, 0) : RollingMethods.roll(iRolls, iFaces, keep);
```
Comment (thpr) [inline code/src/java/pcgen/core/RollingMethods.java]: This has to be bigger than our line length limit...

Comment (grimreaper) [inline code/src/java/pcgen/core/RollingMethods.java]: fixed

**Comment by thpr** [comment]
Still think this file needs reformatting to 90 line length limit

**Comment by grimreaper** [comment]
as discussed the line length was different among different config files. Reformatted the /entire/ file. Leme know if anything else in my IDE should be changed.

**Comment by thpr** [comment]
Now we seem to have line ending differences.... :/

**Comment by grimreaper** [comment]
ugh, I'll fix this later. This is one of the many reasons I'd really like to have a proper entry in gitattributes.

**Comment by thpr** [comment]
how is text=auto not "proper"?  It's there, you may need to ensure your local tools respect it...

**Pull request closed without merging**